tasks/21_check-letters/check-letters.js

```diff
@@ -0,0 +1,9 @@
+export function checkLetters([firstArray, secondArray]) {
+  return secondArray
+      .toLowerCase()
+      .split('')
+      .reduce((acc, item) => {
+        return acc && (firstArray.toLowerCase().indexOf(item) !== -1);
```

Comment:
Extract `firstArray.toLowerCase().indexOf(item) !== -1` to a function with a suitable name. For example `!isItemContainedInArray(item, firstArray)`

Reply:
Rename item to character